samples/gen-go-blog/server/router.go

```diff
@@ -157,7 +170,8 @@ func withMiddleware(serviceName string, router http.Handler, m []func(http.Handl
 		handler = m[i](handler)
 	}
 	handler = TracingMiddleware(handler)
-	handler = PanicMiddleware(handler)
+	// logs any panics encountered in our middleware
+	handler = PanicMiddleware(handler, false)
```

Comment:
Why should we keep this middleware here is the one above does the same stuff?

Reply:
This outer middleware provides value by logging panics from any other middlewares. I think it's worth having given the dependency injection pattern for user defined middleware